packages/message-system/src/config/config.v1.json

```diff
@@ -72,6 +46,73 @@
                 ]
             }
         },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=23.5.1",
+                        "mobile": "!",
+                        "web": "!"
+                    }
+                }
+            ],
+            "message": {
+                "id": "cjdnfj62-b796-4c67-abcb-ac38c15ae343",
+                "priority": 50,
+                "dismissible": false,
+                "variant": "warning",
+                "category": "context",
+                "content": {
+                    "en-GB": "Coinjoin is in public beta. Your feedback is appreciated",
+                    "en": "Coinjoin is in public beta. Your feedback is appreciated",
+                    "es": "Coinjoin está en fase beta pública. Agradecemos tus comentarios.",
+                    "cs": "Coinjoin je ve veřejné beta verzi. Vaše zpětná vazba je vítána.",
+                    "ru": "Coinjoin находится в стадии публичной бета-версии. Мы будем признательны за ваши отзывы.",
+                    "ja": "Coinjoinはパブリックベータ版です。あなたのフィードバックをお待ちしています。"
+                },
+                "context": { "domain": "accounts.coinjoin" },
+                "cta": {
+                    "action": "external-link",
+                    "link": "https://satoshilabs.typeform.com/coinjoin",
+                    "label": {
+                        "en-GB": "Give Feedback",
+                        "en": "Give Feedback",
+                        "es": "Envía tus comentarios",
+                        "cs": "Ohodnotit",
+                        "ru": "Оставить отзыв",
+                        "ja": "フィードバックをする"
+                    }
+                }
+            }
+        },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=23.5.1",
+                        "mobile": "!",
+                        "web": ">=23.5.1"
+                    }
+                }
+            ],
+            "message": {
```

Comment:
validation failed, message has to have `content` https://github.com/trezor/trezor-suite/actions/runs/4960816845/jobs/8876813361?pr=8450#step:11:13

Reply:
bfc55e552d0b9f0bffd3984941ca5d1a29c4bf76